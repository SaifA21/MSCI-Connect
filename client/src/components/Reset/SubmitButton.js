import * as React from 'react';
import {Button } from '@material-ui/core';


const SubmitButton = (props) => {

    return(
        <Button variant='contained' onClick = {props.handleSubmit}>Reset</Button>
    )
  
  }


  export default (SubmitButton)

