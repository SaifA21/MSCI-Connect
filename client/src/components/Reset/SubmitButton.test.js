import renderer from 'react-test-renderer';
import SubmitButton from './SubmitButton';


// Unit Test: Theveenan Nirmalan
describe('submit button', () => {

    it('renders correctly', () => {
        const tree = renderer
          .create(<SubmitButton></SubmitButton>)
          .toJSON();
        expect(tree).toMatchSnapshot();
      });


})