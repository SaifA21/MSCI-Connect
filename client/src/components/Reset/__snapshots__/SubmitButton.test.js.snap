// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`submit button renders correctly 1`] = `
<button
  className="MuiButtonBase-root MuiButton-root MuiButton-contained"
  disabled={false}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="MuiButton-label"
  >
    Reset
  </span>
</button>
`;
